src/main/resources/committers-names.properties

```diff
@@ -0,0 +1,22 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# additional names used on ML by committers
+
+brianf = Brian Fox
+cstamas = Tamás Cservenák
+hboutemy = Hervé Boutemy
```

Comment:
isn't this in parent POM?

Reply:
I reade names from - https://projects.apache.org//json/foundation/people_name.json
But someone can use a little different name according to national chars and so ...

Reply:
haha US system. be sure dates have correct format :)